cli/src/main/java/io/specmesh/cli/Main.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2023 SpecMesh Contributors (https://github.com/specmesh)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.specmesh.cli;
+
+import picocli.CommandLine;
+
+/** SpecMesh CLI client */
+public final class Main {
+    private Main() {}
+
+    /**
+     * Main cli
+     *
+     * @param args - set of CLI args for processing
+     */
+    public static void main(final String[] args) {
+
+        new CommandLine(new KafkaProvisionCommand()).execute(args);
+    }
+}
```

Comment:
I *think* you need to do more to be able to output help.  Would need to read the docs again to work it out.